skietbaan_fe/package.json

```diff
@@ -14,6 +15,7 @@
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^5.0.0-alpha.8",
     "react-scripts": "^1.1.5",
+    "react-select": "^2.4.0",
```

Comment:
I would suggest to rather stay fixed on a version, or at least only be able to update the patch version only e.g. ~2.4.0

Reply:
We are staying on this version

Reply:
If you want to keep this fixed then remove the `^` that indicates that it may be updated automatically